### PR TITLE
[docs] introduce a Dockerfile for the website

### PR DESCRIPTION
**Pull request by efx**
Happy to document this where appropriate. In the meantime, here are a few notes for using on a Unixy system where you have Docker installed.

## instructions

1. From the root of this repository, build the docker image:
```sh
 docker build . -t exercism-node:v1
```
2. Run the container that has everything configured:
```
docker run -it --rm --name exercism-website -p 3005:3005 -v "$PWD":/usr/src/app exercism-node:v1
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM node:13
```
Comment (SleeplessByte) [inline Dockerfile:1]: Don't use unstable node (all uneven numbers are inherently not stable and will _never_ reach LTS). Instead use lts, or use debian and install node.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+# https://github.com/docsifyjs/docsify-cli/issues/78
+RUN apt-get update \
+	&& apt-get install dos2unix \
+	&& dos2unix /usr/local/lib/node_modules/docsify-cli/bin/docsify
+
+EXPOSE 3005
+USER node
+
+WORKDIR /usr/src/app
```
Comment (SleeplessByte) [inline Dockerfile:12]: UNIX paths IIRC do have some sanity in their naming. Shouldn't this be places in `/var/something` ?

Comment (efx) [reply Dockerfile:12]: I would disagree actually. Most of the node images use `/usr/src/app` so it is more a convention in node docker documentation: https://github.com/nodejs/docker-node/blob/5351774b970cd32b80fc0c47a8abff5ba155fccc/docs/BestPractices.md

Comment (SleeplessByte) [reply Dockerfile:13]: `/usr/local/bin` is something _very_ different than `/usr/src/app`. This is, on standard UNIX systems, a directory for programs that a normal user may run. Putting binaries in there that the resulting node image can "run" is therefore 'compliant'.

Note: I don't actually know what the _right_ place is to put the source for the directory, and if  `/usr/src/ ` is a convention I simply don't know about, then by all means, let's keep it there. I know this is pedantic, and I'm not saying this to annoy you, but I don't want to ever have to answer the question "why is it in `/usr/src/app`?"

I'll ask our devops peeps who live and breath UNIX.

Comment (efx) [reply Dockerfile:13]: @SleeplessByte can we continue this conversation on Slack sometime? I think I am missing your concerns.

Comment (SleeplessByte) [reply Dockerfile:13]: Always! Ping me whenever you like :)

Comment (ErikSchierboom) [reply Dockerfile:13]: I was wondering if you've (@efx and @SleeplessByte) managed to come to a conclusion?

Comment (efx) [reply Dockerfile:13]: @ErikSchierboom @SleeplessByte sorry; I have not had the stamina to approach this. I am blocked on understanding *why* specifying the mount point is a problem and would like to understand the object better before changing or merging. Happy to flesh this out here too.

Comment (ErikSchierboom) [reply Dockerfile:13]: No worries. I'll leave it up to @SleeplessByte to check on this.

**Comment by ErikSchierboom** [comment]
@efx Could you add instructions to the `docs/maintainers/website.md` document on how to use the Docker image?

File: Dockerfile (hunk 2 above)
Comment (efx) [reply Dockerfile:12]: We met and discussed and decided to put within `/opt/<name-of-project>` per our reading of the [FHS standard](https://refspecs.linuxfoundation.org/FHS_2.3/fhs-2.3.html#OPTADDONAPPLICATIONSOFTWAREPACKAGES).

**Pull request merged**